tests/test_recipes.py

```diff
@@ -268,8 +272,13 @@ def test_chunks(
     xr.testing.assert_identical(ds_actual, ds_expected)
 
 
-def test_lock_timeout(netCDFtoZarr_sequential_recipe, execute_recipe):
-    RecipeClass, file_pattern, kwargs, ds_expected, target = netCDFtoZarr_sequential_recipe
+def test_lock_timeout(netCDFtoZarr_recipe, execute_recipe):
+    RecipeClass, file_pattern, kwargs, ds_expected, target = netCDFtoZarr_recipe
+
+    # `netCDFtoZarr_recipe` fixture is parametrized. We don't need to run this test more than once.
+    if len(file_pattern.merge_dims) != 0:
+        pytest.skip("It's redundant to run this test more than once.")
```

Comment:
In general it's not redundant to run the tests more than once. Each `netCDFtoZarr_recipe` instance is different. Running the same test on different inputs usually gives us better coverage. We are skipping here  because we think there are actually not meaningful differences between each parametrized run; that probably indicates that we are using the wrong fixtures in the first place.